### PR TITLE
HDDS-1462. Fix content and format of Ozone documentation

### PR DESCRIPTION
**Pull request by elek**
During the review of HDDS-1457 I realized that the current documentation contains many outdated information regarding the usage of docker, build commands or s3 usage.

The security information is also rendered in an incorrect way.

The png files for the prometheus page are missing (were included in the patch of HDDS-846 but missing from the commit).

See: https://issues.apache.org/jira/browse/HDDS-1462

### CODE REVIEW

**Comment by ajayydv** [comment]
+1 with branch conflicts resolved.

File: hadoop-hdds/docs/content/OzoneFS.md [hunk 1]
```diff
@@ -122,4 +122,7 @@ Hadoop version | Required jar            | OzoneFileSystem implementation
 3.2            | filesystem-lib-current  | org.apache.hadoop.fs.ozone.OzoneFileSystem
 3.1            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.OzoneFileSystem
 2.9            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
-2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
\ No newline at end of file
+2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
```
Comment (arp7) [inline hadoop-hdds/docs/content/OzoneFS.md]: @ajayydv you probably didn't mean to leave the change markers in place here.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 0 | Docker mode activated. |
| -1 | patch | 8 | https://github.com/apache/hadoop/pull/767 does not apply to trunk. Rebase required? Wrong Branch? See https://wiki.apache.org/hadoop/HowToContribute for help. |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/767 |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-767/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/docs/content/OzoneFS.md (hunk 1 above)
Comment (ajayydv) [reply hadoop-hdds/docs/content/OzoneFS.md]: thanks for catching this, removed!!

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 0 | Docker mode activated. |
| -1 | patch | 6 | https://github.com/apache/hadoop/pull/767 does not apply to trunk. Rebase required? Wrong Branch? See https://wiki.apache.org/hadoop/HowToContribute for help. |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/767 |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-767/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/docs/content/S3.md [hunk 2]
```diff
@@ -83,16 +83,37 @@ Endpoint                            | Status          | Notes
...
+awsSecret=c261b6ecabf7d37d5f9ded654b1c724adac9bd9f13e247a235e567e8296d2999
+
+```
+
+Now, you can use the key and the secret to access the S3 endpoint:
+
+```
+export AWS_ACCESS_KEY_ID=testuser/scm@EXAMPLE.COM
+export AWS_SECRET_ACCESS_KEY=c261b6ecabf7d37d5f9ded654b1c724adac9bd9f13e247a235e567e8296d2999
+aws s3api --endpoint http://localhost:9878 create-bucket --bucket bucket1
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/S3.md]: should this be --endpoint-url instead of --endpoint?

Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/S3.md]: we may change this to use aws s3api like below:

aws configure set default.s3.signature_version s3v4
aws configure set region us-west-1
aws configure set aws_access_key_id testuser/om@EXAMPLE.COM
aws configure set aws_secret_access_key 95fdf9ada7c312faf42cf774fc9f9ff0fd6be4a94477ca9a34ee417931928183

**Pull request merged**